pkg/exporter/exporter.go

```diff
@@ -32,9 +32,13 @@ const (
 )
 
 func ruleToResult(ruleID string) string {
-	return fmt.Sprintf("preflight_%s", strings.ReplaceAll(ruleID, ".", "_"))
+	return strings.ReplaceAll(ruleID, ".", "_")
```

Comment:
Isn't this unnecessary now as rule IDs are required to be valid Rego names?

Reply:
They are not strictly required though.

We check that in the linter, which flags recommendations but, at least for now, we don't impose those rules.

If at some point we want to impose them, we would run those checks at the beginning of the execution of Preflight.

Reply:
Yeah, I feel that we should be enforcing it. I think packages should be linted when they are loaded as part of a `preflight check` and packages that fail should be ignored. This PR is probably not the place to make that change though.